test-app/tests/integration/components/file-input-field-test.gts

```diff
@@ -57,6 +57,8 @@ module('Integration | Component | Fields | FileInput', function (hooks) {
     </template>);
 
     assert.dom('[data-label]').hasText('Label');
+    assert.dom('label').hasClass('text-body-and-labels');
```

Comment:
FileInput is a bit different than the others, where because of the markup, `data-label` is actually a nested element within `<label`.  Due to that, I'm targeting the `label` tag directly here instead of using `[data-label]`